packages/squiggle-lang/src/public/SqValue/index.ts

```diff
@@ -153,6 +160,13 @@ export class SqLambdaValue extends SqAbstractValue<"Lambda", SqLambda> {
   asJS() {
     return this.value; // SqLambda is nicer than internal Lambda, so we use that
   }
+
+  toCalculator(): SqCalculatorValue | undefined {
```

Comment:
I used this for testing, but we're not using it otherwise right now. But it seems good to have.